src/eradius_lib.erl

```diff
@@ -469,7 +469,7 @@ salt_crypt(Op, SharedSecret, B, <<PlainText:16/binary, Remaining/binary>>, Ciphe
 	    end,
     salt_crypt(Op, SharedSecret, Bnext, Remaining, <<CipherText/binary, NewCipherText/binary>>);
 
-salt_crypt(Op, _SharedSecret, _B, << >>, CipherText) ->
+salt_crypt(_Op, _SharedSecret, _B, << >>, CipherText) ->
```

Comment:
I don't mind that this warning fix goes into the same pull request, but could you please make it a separate commit?